### PR TITLE
Fix #270: Update Jib Core to v0.16.0

### LINKED ISSUE #270: Update JIB dependency to latest release
### Description

Upgrade JIB (jib-core) dependency to [latest](https://github.com/GoogleContainerTools/jib/releases/latest) available release (as of now [`0.15.0`](https://github.com/GoogleContainerTools/jib/releases/tag/v0.15.0-core))

- [x] Submit CQ for dependency

### Related topics

- https://github.com/GoogleContainerTools/jib/issues/1555

### PR DESCRIPTION
**Pull request by rohanKanojia**
## Description

Fix #270: Update Jib Core to v0.16.0

- [x] CQ for Jib-Core 0.16.0 https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22760
- [x] CQ For Apache HttpClient 4.5.10 https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22761
## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [X] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [X] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [X] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [X] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

File: jkube-kit/build/service/jib/src/main/java/org/eclipse/jkube/kit/service/jib/JibServiceUtil.java [hunk 1]
```diff
@@ -118,7 +118,7 @@ public static void buildContainer(JibContainerBuilder jibContainerBuilder, TarIm
     public static JibContainerBuilder containerFromImageConfiguration(
         ImageConfiguration imageConfiguration, Credential pullRegistryCredential) throws InvalidImageReferenceException {
         final JibContainerBuilder containerBuilder = Jib.from(getRegistryImage(getBaseImage(imageConfiguration), pullRegistryCredential))
-                .setFormat(ImageFormat.OCI);
+                .setFormat(ImageFormat.Docker);
```
Comment (manusa) [inline jkube-kit/build/service/jib/src/main/java/org/eclipse/jkube/kit/service/jib/JibServiceUtil.java:121]: What's the purpose of this change?

Comment (rohanKanojia) [reply jkube-kit/build/service/jib/src/main/java/org/eclipse/jkube/kit/service/jib/JibServiceUtil.java:121]: Not sure why it was working in previous versions. If I set image format to `ImageFormat.OCI` then the generated tarball can't be loaded by docker when using `docker load -i` step.

Comment (manusa) [reply jkube-kit/build/service/jib/src/main/java/org/eclipse/jkube/kit/service/jib/JibServiceUtil.java:121]: :+1: 
Maybe they updated the OCI spec version or added incompatible entries.

**Review by manusa — APPROVED** [review]
